app/views/static_pages/home.html.erb

```diff
@@ -16,8 +16,14 @@
         </div>
 
 		<div class="row galleries">
-	   		<div class="gallery">
-				<a href="http://www.mayflowerhomes.com/calendar/" target="_blank">
+	        	<div class="gallery">
+    				<div class="gallery-default_background">
+						<%= link_to image_tag("mayflower_logo_large.png"), resources_library_path %>}
+					</div>
+           			<h2 class="info">Mayflower Library</h2></a>
```

Comment:
This text is not linked, unlike the other image descriptions on the resources page.